controllers/tags.js

```diff
@@ -0,0 +1,67 @@
+const TagModel = require("../models/tags");
+
+/**
+ * Creates a new tag
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const addTag = async (req, res) => {
+  try {
+    const { id, tagData } = await TagModel.addTag(req.body);
```

Comment:
Are we sending createdBy in the body only? Can we use the authenticate middleware here to get the user data and put that in the createdBy field?

Reply:
done